src/markup.c

```diff
@@ -28,6 +25,35 @@ char *markup_quote(char *str)
         return str;
 }
 
+static
+char *markup_unquote(char *str)
+{
+        if (str == NULL) {
```

Comment:
For the static functions, sure.  But my understanding of the rest of the code base isn't complete enough for me to be confident that we don't depend on passing NULL to markup_strip or markup_transform.